packages/connector-wechat-native/README.md

```diff
@@ -406,7 +406,77 @@ func application(_ app: UIApplication, open url: URL, options: /*...*/) -> Bool
 
 ### Android
 
-TBD
+我们假设你已经在你的应用中集成了 [Logto Android SDK](https://docs.logto.io/docs/recipes/integrate-logto/android)。之后的流程很简单，你甚至不需要阅读微信 SDK 文档：
+
+**1. 添加 `Wechat Open SDK` 到你的项目中**
+
+确保 `mavenCentral()` 已经被添加至你 Gradle 项目中的 repositories 中:
+
+```kotlin
+repositories {
+  // ...
+  mavenCentral()
+}
+```
+
+添加 Wechat Open SDK 依赖:
+
+```kotlin
+dependencies {
+  // ...
+  api("com.tencent.mm.opensdk:wechat-sdk-android:6.8.0")  // kotlin-script
+  // 或
+  api 'com.tencent.mm.opensdk:wechat-sdk-android:6.8.0'   // groovy-script
+}
+```
+
+**2. 将 `WXEntryActivity` 引入到项目中**
+
+在项目的根 package 下创建一个 `wxapi` package，并在`wxapi` package 中创建 `WXEntryActivity`（以 `com.sample.app` 为例）：
```

Comment:
```suggestion
在项目的根 package 下创建一个 `wxapi` package，并在`wxapi` package 中创建 `WXEntryActivity`。
以 `com.sample.app` 为例）：
```

Feels like the colon does not belong to the whole sentence at the front.   Should come with the content in bracket only.